.gitignore

```diff
@@ -2,3 +2,5 @@
 *.pem
 *.env
 *.ini
+!digitalocean.ini.example
+!pihole.env.example
```

Comment:
I think it is best to have *.example in secrets directory. Howevet, if we don’t ignore the rest of the files, there is a good chance and .ini or another file will be committed in this repo

Reply:
When you add any file in this directory, git status command shouldn’t see those new files as added new files

Reply:
Thanks for the review, I will make the corrections asap.